env.kesch.sh

```diff
@@ -236,6 +236,16 @@ EOF
 EOF
         export FC=gfortran
         ;;
+    pgi ) 
+        cat > $ENVIRONMENT_TEMPFILE <<-EOF
+            # Generated with the build script
+            # implicit module purge
+            module load craype-haswell
+            module load GCC/4.9.3-binutils-2.25
+            module load PrgEnv-pgi/16.3
+EOF
+        export FC=mpif90
+        ;;	
```

Comment:
Do we also need to adapt `get_fcompiler_cmd()` to pgi? Or does this work already?